examples/build/README.md

```diff
@@ -0,0 +1,6 @@
+## Build Example
+
+This folder holds a super simple devcontainer configuration that builds a local Dockerfile with a devcontainer feature. You can start this project via:
```

Comment:
where is the devcontainer feature though? 🤔

Reply:
ah I was adding / removing it to test build times for caching. I must have forgot to add it back! I'll do it now